restli-tools/src/main/resources/macros/library.vm

```diff
@@ -13,6 +13,15 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 *#
+#macro(parseMethodsFromTemplates)
+  ## Action methods
+  #parse("apiVmTemplates/action.vm")
+  ## CRUD methods
+  #foreach($method in $spec.restMethods)
+      #parse("apiVmTemplates/rest.${method.method}.vm")
+  #end
+#end
```

Comment:
Need to include finders and batch finders if the other PR goes first.

Reply:
I have reviewed the other Finder PR.
I recommend this PR go first.
Then I will help resolving conflicts for that PR.
Then I will do another patch to add interface for finders.